multicluster/controllers/multicluster/commonarea/resourceimport_controller.go

```diff
@@ -325,12 +332,14 @@ func (r *ResourceImportReconciler) handleResImpDeleteForEndpoints(ctx context.Co
 			Namespace: resImp.Spec.Namespace,
 		},
 	}
-	err := r.localClusterClient.Delete(ctx, ep, &client.DeleteOptions{})
+	err := client.IgnoreNotFound(r.localClusterClient.Delete(ctx, ep, &client.DeleteOptions{}))
 	if err != nil {
-		klog.InfoS("Failed to delete imported Endpoints", "endpoints", epNamespacedName, "err", err)
-		return ctrl.Result{}, client.IgnoreNotFound(err)
+		klog.ErrorS(err, "Failed to delete imported Endpoint", "endpoint", epNamespacedName)
```

Comment:
Here we are referring to K8s's resource definition `Endpoints`.
```suggestion
		klog.ErrorS(err, "Failed to delete imported Endpoints", "endpoints", epNamespacedName)
```

Reply:
Fixed